pages/api/helpjuice/search.page.ts

```diff
@@ -0,0 +1,23 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import fetch from 'node-fetch';
+
+const proxySearchParent = async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+): Promise<void> => {
+  try {
+    const searchTerm = req.query.query;
```

Comment:
Not required, but you could also write it like this
```suggestion
    const { query: searchTerm } = req.query;
```